quark/metrics.q

```diff
@@ -40,6 +40,12 @@ namespace mdk_metrics {
         @doc("Map destination node UUID to success=1/failure=0 of interaction.")
         Map<String,int> results = {};
 
+        @doc("Causal clock level at start.")
```

Comment:
Yeah, you'd want to add another column. We want to store both the clock level of the start of the interaction and the clock level at the end of the interaction so we correlate with logs on both start and end.